Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -957,23 +957,12 @@ private extension PurchasesOrchestrator {
             case let .failure(error):
                 let purchasesError = error.asPublicError
 
-                #if swift(>=5.6)
-                @MainActor
-                func complete() {
-                    completion?(transaction, nil, purchasesError, false)
-                }
-                #else
-                @Sendable
-                @MainActor
-                func complete() {
-                    completion?(transaction, nil, purchasesError, false)
-                }
-                #endif
-
                 if finishable {
-                    self.finishTransactionIfNeeded(transaction) { complete() }
+                    self.finishTransactionIfNeeded(transaction) {
+                        completion?(transaction, nil, purchasesError, false)
+                    }
                 } else {
-                    complete()
+                    completion?(transaction, nil, purchasesError, false)
```

Comment:
honestly this is easier to read too

Reply:
Yeah totally 🤷🏻‍♂️ not sure why I didn't realize it earlier that we could go back to this.